### DISCUSSION TITLE
rules on after reset 1 ?

### DISCUSSION THREAD

**Discussion by DustinThal** [arendst/Tasmota#17000] [Q&A] [answered]
Hi, 
i there a line in some of the files to activate rules as standard.
S, that all rules are on after a reset 1?

BR Dustin

**Comment by sfromis** [comment, marked as answer]
When building with `#define USER_RULE1`, 2 or 3 for the rule set text, you can also have commands in `#define USER_BACKLOG`, like to activate rule sets.

**Reply by DustinThal** [reply]
best thanks that saved my day
